cmd/govim/oswatcher_darwin.go

```diff
@@ -0,0 +1,63 @@
+// +build darwin
+
+package main
+
+import (
+	"fmt"
+	"path/filepath"
+	"time"
+
+	"github.com/fsnotify/fsevents"
+)
+
+const fRemoved = fsevents.ItemRemoved | fsevents.ItemRenamed
+const fChanged = fsevents.ItemCreated | fsevents.ItemModified | fsevents.ItemChangeOwner
+
+func newOSWatcher(gomodpath string) (*osWatcher, error) {
+	dirpath := filepath.Dir(gomodpath)
+	dev, err := fsevents.DeviceForPath(dirpath)
+	if err != nil {
+		return nil, fmt.Errorf("failed to retrieve device for path %v: %v", dirpath, err)
+	}
+
+	es := &fsevents.EventStream{
+		Paths:   []string{dirpath},
+		Latency: 200 * time.Millisecond,
```

Comment:
This is, I'm guessing a throttle on the events? Why set any value?

Reply:
From the [docs](https://developer.apple.com/documentation/coreservices/1447341-fseventstreamcreaterelativetodev):
> **latency**
The number of seconds the service should wait after hearing about an event from the kernel before passing it along to the client via its callback. Specifying a larger value may result in more effective temporal coalescing, resulting in fewer callbacks.

So I went for an arbitrary value. The example in their repo uses 500ms. Setting it (close) to zero could generate lots of callbacks and unnecessary overhead. But we might go for a lower value and increase it if anyone experience issues?

Reply:
Sounds good.